test/spec/browser/request-strategy/only-JSONP-when-XHR-failure.js

```diff
@@ -1,63 +0,0 @@
-'use strict';
```

Comment:
Yeah, because it seems a tricky behavior so a test should help it debug/no break it